catalog/CSIC/CSIC_SPEI_2_8.jsonnet

```diff
@@ -0,0 +1,330 @@
+local id = 'CSIC/SPEI/2_8';
+local subdir = 'CSIC';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
```

Comment:
please add
'gee:skip_indexing': true
at the top level.

It's a temporary field that's required for now until we improve a few internal steps. We will remove it when the dataset is ready to go.